src/test/java/nextstep/courses/domain/ImageTest.java

```diff
@@ -0,0 +1,29 @@
+package nextstep.courses.domain;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static nextstep.courses.domain.ImageType.JPG;
+import static org.assertj.core.api.Assertions.assertThatNoException;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+public class ImageTest {
+
+    @Test
+    @DisplayName("1MB가 넘는 이미지는 생성할 수 없다.")
+    void checkSize(){
+        assertThatThrownBy(() -> new Image(3 * 1024 * 1024, JPG, 300, 200)).isInstanceOf(IllegalArgumentException.class);
```

Comment:
```suggestion
        assertThatThrownBy(() -> new Image(3145728, JPG, 300, 200)).isInstanceOf(IllegalArgumentException.class);
```

저는 `3 * 1024 * 1024`이걸 보자마자 머릿속이 복잡해집니다.
수식이 중요한가요?

무엇을 표현하고 싶은건가요?
> 코드는 사람이 구현하고 사람이 보는 영역입니다

Reply:
Byte -> MB로 나누지 않고도 3MB란걸 보여주기 위해 사용했는데, 더 알기 쉽게 변수를 사용해보겠습니다 :)